contrib/babelfishpg_tsql/src/dbcmds.c

```diff
@@ -225,6 +225,7 @@ gen_dropdb_subcmds(const char *schema,
 	}
 	/* Then drop db_owner and dbo in that order */
 	appendStringInfo(&query, "DROP OWNED BY dummy, dummy CASCADE; ");
```

Comment:
I'm confused why we use dummy in this case ?

Reply:
just a placeholder, we replace it with the correct value after the statement has been converted into a node